src/libs/ajax/data-table-providers/DataTableProvider.ts

```diff
@@ -125,6 +133,7 @@ export interface DataTableFeatures {
   supportsTypeRenaming: boolean;
   supportsEntityRenaming: boolean;
   supportsEntityUpdating: boolean;
+  supportEntityUpdatingTypes?: string[];
```

Comment:
Super nit: the other fields here are named "supports..." vs "support...".

Reply:
ah man, thank you for noticing that - if we didnt catch it, it would make so mad :D